src/index/base.cpp

```diff
@@ -63,30 +63,31 @@ bool BaseIndex::Init()
     if (locator.IsNull()) {
         m_best_block_index = nullptr;
     } else {
-        m_best_block_index = g_chainman.m_blockman.FindForkInGlobalIndex(::ChainActive(), locator);
+        m_best_block_index = m_chainstate->m_blockman.FindForkInGlobalIndex(m_chainstate->m_chain, locator);
     }
-    m_synced = m_best_block_index.load() == ::ChainActive().Tip();
+    CChain& active_chain = m_chainstate->m_chain;
```

Comment:
In commit "index: Add chainstate member to BaseIndex" (0945b4117e0209d156a6b74c407f4d32e0cf2c1a)

Seems like active_chain could be used above too. Fine either way in case this was intentional, though.